sdk/servicebus/service-bus/src/util/connectionStringUtils.ts

```diff
@@ -98,5 +104,11 @@ export function parseServiceBusConnectionString(
     output.sharedAccessKey = parsedResult.SharedAccessKey;
     output.sharedAccessKeyName = parsedResult.SharedAccessKeyName;
   }
+  if (parsedResult.UseDevelopmentEmulator) {
+    output.useDevelopmentEmulator = Boolean(parsedResult.UseDevelopmentEmulator);
+  } else {
+    output.useDevelopmentEmulator = false;
+  }
```

Comment:
nitpick: this should be equivalent, no?
```suggestion
    output.useDevelopmentEmulator = Boolean(parsedResult.UseDevelopmentEmulator);
```

Reply:
yes it's equivalent and simpler